dts/arm/st/u5/stm32u5.dtsi

```diff
@@ -547,6 +547,7 @@
 			interrupts = <113 0>;
 			status = "disabled";
 			#io-channel-cells = <1>;
+			vref-mv = <3000>;
 			has-temp-channel;
```

Comment:
The pb here is that TS CAL1 & TS CAL2  are calibration values acquired by ADC 1 with a VREF+ = 3V and the stm32U5 boards (nucleo as well as disco kit) have a VRef at 3.3V
Should it be ` ts-cal-vrefanalog = <3000>` and `vref-mv = <3000>;` ?
Again, the st,stm32-adc.yaml  defined the vref-mv as "the reference voltage of the ADC in mV (on the target board)." ==> better  set that value by the board DTS (not the SOC).

Reply:
Second the ADC4 has 12 bits resolution and ADC1 has 14. The TS_CAL1 and TS_CAL2 are acquired with ADC1 on 14bits.